### DISCUSSION TITLE
Where does the name Mangum come from?

### DISCUSSION THREAD

**Discussion by lesiki** [Kludex/mangum#284] [Q&A] [answered]
I've been using this for a couple of years now and only realised today that it's Mangum and not "Magnum" 😬

**Comment by jordaneremieff** [comment, marked as answer]
Yeah can be a bit confusing.

It is named after one of my favourite musicians, [Jeff Mangum](https://en.wikipedia.org/wiki/Jeff_Mangum). Initially I was just trying to share some proof of concept Gist and chose the name without much consideration.

**Reply by lesiki** [reply]
Nice, I'll be checking out some Mangum tunes tonight
